consensus/consensus.go

```diff
@@ -145,4 +149,9 @@ type PoSA interface {
 	EnoughDistance(chain ChainReader, header *types.Header) bool
 	IsLocalBlock(header *types.Header) bool
 	AllowLightProcess(chain ChainReader, currentHeader *types.Header) bool
+	GetHighestJustifiedHeader(chain ChainHeaderReader, header *types.Header) *types.Header
```

Comment:
Prefer `GetJustifiedHeader` than `GetHighestJustifiedHeader`,  when we mean `JustifiedHeader`, of course mean the latest one.

Reply:
fixed